site-manage/datemanager/impl/src/main/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java

```diff
@@ -802,7 +802,7 @@ public DateManagerValidation validateGradebookItems(String siteId, JSONArray gra
 							}
 						}
 					}
-					gbitem = gradingService.getAssignmentById(groupId, itemId);
+					gbitem = gradingService.getAssignment(groupId, getCurrentSiteId(), itemId);
```

Comment:
if `getAssignmentById` doesn't work, you need to get rid of it!

Reply:
Cannot be deleted, it is used elsewhere in the code.

Reply:
where?

Reply:
https://github.com/sakaiproject/sakai/blame/master/gradebookng/impl/src/main/java/org/sakaiproject/grading/impl/GradingServiceImpl.java#L5290